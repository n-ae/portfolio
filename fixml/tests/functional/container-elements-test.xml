<root>
  <section>
    <item>A</item>
    <item>B</item>
  </section>
  <section>
    <item>C</item>
    <item>A</item>
  </section>
  <group>
    <member>X</member>
  </group>
  <group>
    <member>Y</member>
  </group>
</root>