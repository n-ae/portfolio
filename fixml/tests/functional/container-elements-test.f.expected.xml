<?xml version="1.0" encoding="utf-8"?>
<root>
  <section>
    <item>A</item>
    <item>B</item>
  </section>
  <section>
    <item>C</item>
  </section>
  <group>
    <member>X</member>
  </group>
  <group>
    <member>Y</member>
  </group>
</root>
